### ISSUE TITLE
Link to WikiReader Plugin

### ISSUE THREAD

**Issue by tillmannheigel** [Bartvelp/zim-converter#2]
Hey,

I am super curious about your approach of extending KOReader to read from SQLite databases. You are mentioning a WikiReader plugin in the README.md, but I could not find it. Did you already publish it?

Greetings, Tillmann

**Comment by Bartvelp** [comment]
Hi Tillman,

The WikiReader plugin is available in this pull request: https://github.com/koreader/koreader/pull/9534
There really is not to much to it, checkout this file for the Sqlite connection: https://github.com/koreader/koreader/blob/db17e74382f47aa22f2e3d5bd2002ea0e328592b/plugins/wikireader.koplugin/reader-widget.lua

**Comment by tcsenpai** [comment]
I am really interested in checking this out too. I see the pr is very old. Right now what should I do to "simply" install your plugin into koreader? Anything else than just copying the plugin folder into koreader? I see there are other files in the commit so i am wondering what do i have to do

**Comment by Bartvelp** [comment]
Yes I believe you need to copy the plugin folder and that will give most of the functionality, I also hacked the link implementation so you can click inter-wikipedia articles, I think that is the only other patch.

You also need a Wikipedia DB, you can download the one from the readme of this project.

**Comment by tcsenpai** [comment]
I will open an issue on its own as I found a weird behavior on my Kobo. I'll briefly investigate and I'll open it
